crates/lib/src/api.rs

```diff
@@ -292,7 +292,7 @@ impl From<ControllerJobExecutionResult> for ExecutionResults {
 ///
 /// # Errors
 ///
-/// May error if a [`gRPC`] client cannot be constructed, or a [`gRPC`]
+/// May error if a [`Qcs`] client cannot be constructed, or if the ``gRPC``
 /// call fails.
```

Comment:
Shouldn't this be \`gRPC\`?

Alternatively, assuming that clippy flagged this in the first place, create a `clippy.toml` file with the [following contents](https://rust-lang.github.io/rust-clippy/master/index.html#doc_markdown):

```toml
doc-valid-idents = ["..", "gRPC"]
```

Reply:
I went with double backticks because that is how it was done in other places. It doesn't appear to make a difference in the rendered docs, so I updated them all to single backticks.